### DISCUSSION TITLE
绘制的点线面样式怎么修改

### DISCUSSION THREAD

**Discussion by lhw-888** [zouyaoji/vue-cesium#289] [Q&A]
![image](https://user-images.githubusercontent.com/88429025/196114755-85d916c9-8fba-4d69-9137-1a645960a57c.png)
这个参数传的参数是啥

**Comment by zouyaoji** [comment]
你截图的是，整体改的方式，可配置项见这儿：

https://github.com/zouyaoji/vue-cesium/blob/dev/packages/composables/use-drawing/defaultOpts.ts#L204

![image](https://user-images.githubusercontent.com/24430116/196345917-2fab5dc7-958a-4283-af7f-3a932d45d11d.png)

**Reply by zouyaoji** [reply]
如果是想对每一个点、线、面单独设置的话，
通过模板引用来获取到绘制的数组
drawingsRef.getDrawingActionInstances()[3].cmpRef.value.renderDatas

**Reply by lhw-888** [reply]
> getDrawingActionInstances

好勒，谢谢解答

**Reply by lhw-888** [reply]
乔巴船长，还有个问题 
![image](https://user-images.githubusercontent.com/88429025/196357096-70063c8f-ed64-4fc9-be70-1ab40c74e3b3.png)
这个线的颜色怎么改，

**Reply by zouyaoji** [reply]
不难理解哈，应该在primitiveOpts中改。
primitiveOpts 对应组件  https://zouyaoji.top/vue-cesium/#/zh-CN/component/primitives/vc-primitive 
polylineOpts 对应组件  https://zouyaoji.top/vue-cesium/#/zh-CN/component/geometries/vc-geometry-polyline

**Reply by lhw-888** [reply]
ok，可以了，赞

**Reply by zouyaoji** [reply]
好用记得回来 star 一下~
